rand_core/src/lib.rs

```diff
@@ -386,11 +446,29 @@ pub trait SeedableRng: Sized {
     #[cfg(feature = "getrandom")]
     #[cfg_attr(doc_cfg, doc(cfg(feature = "getrandom")))]
     fn from_entropy() -> Self {
-        let mut seed = Self::Seed::default();
-        if let Err(err) = getrandom::getrandom(seed.as_mut()) {
-            panic!("from_entropy failed: {}", err);
+        match Self::try_from_entropy() {
+            Ok(res) => res,
+            Err(err) => panic!("from_entropy failed: {}", err),
         }
-        Self::from_seed(seed)
+    }
+
+    /// Creates a new instance of the RNG seeded via [`getrandom`].
+    ///
+    /// This method is the recommended way to construct non-deterministic PRNGs
+    /// since it is convenient and secure.
```

Comment:
Wording is slightly confusing: both `from_entropy` and `try_from_entropy` are "the recommended way".

Reply:
Fixed. Only `from_entropy` is now listed as "the recommended" with a caveat about panicking.